toplevel/role/role.go

```diff
@@ -230,6 +235,26 @@ func asItems(xs []entry) (items []vault.Item) {
 	return
 }
 
+// validates existence of 'Mount' specified within each desired role
+// nonexistent mount paths are appended w/ culprit role names and returned in an error message
+func validateDesiredRoleMounts(desiredRoles []entry, existingMounts map[string]*api.AuthMount) error {
+	var errSB strings.Builder
```

Comment:
Look at `errors.Join()` available starting from Go **1.20** and newer.

Even though you have a simple case here and don't wrap multiple errors (error types), it would be fine here.